itou/api/applicants_api/serializers.py

```diff
@@ -0,0 +1,65 @@
+from typing import Optional
+
+from rest_framework import serializers
+
+from itou.users.models import User
+
+
+class ApplicantSerializer(serializers.ModelSerializer):
+    """Applicant serializer: job seeker field of a job application"""
+
+    civilite = serializers.SerializerMethodField()
+    nom = serializers.CharField(source="first_name")
+    prenom = serializers.CharField(source="last_name")
+    courriel = serializers.CharField(source="email")
+    telephone = serializers.CharField(source="phone")
+    adresse = serializers.CharField(source="address_line_1")
+    complement_adresse = serializers.CharField(source="address_line_2")
+    code_postal = serializers.CharField(source="post_code")
+    ville = serializers.CharField(source="city")
+    date_naissance = serializers.SerializerMethodField()
+    lieu_naissance = serializers.SerializerMethodField()
+    pays_naissance = serializers.SerializerMethodField()
+    lien_cv = serializers.SerializerMethodField()
```

Comment:
Pourquoi utiliser une méthode ? DRF fait ce qui est attendu avec les attributs (appelle leur `__str__` pour le CharField), tous les tests passent avec la modif suivante :
```suggestion
    lieu_naissance = serializers.CharField(source="birth_place")
    pays_naissance = serializers.CharField(source="birth_country")
    lien_cv = serializers.CharField(source="resume_link")
```